app/controllers/likes_controller.rb

```diff
@@ -1,8 +1,8 @@
 class LikesCountroller < ApplicationController
-  def new
-    @like = Like.new
-    @post = Post.find(params[:post_id])
-  end
+  #   def new
+  #     @like = Like.new
+  #     @post = Post.find(params[:post_id])
+  #   end
```

Comment:
- [ ] Dear @SabaAhmad404, You did well adding the `current_user` method as requested by the previous reviewer. But while doing so I have noticed you have commented out codes in your new changes. Please consider removing `commented codes` as it will make your codebase messy, Remember if you ever like to see the codes you have written once you can use your commit history.

Reply:
@TadesseDev Oh thanks for noticing I forgot to uncomment that method